### PR TITLE
Updated Azure Container Service to Azure Kubernetes Service

### PR DESCRIPTION
**Pull request by VaishaliRaval**
Updated Azure Container Service to Azure Kubernetes Service

### CODE REVIEW

**Comment by VaishaliRaval** [comment]
Fixes https://github.com/dotnet/docs/issues/28047 https://github.com/dotnet/docs/issues/27170

File: docs/architecture/modernize-with-azure-containers/modernize-existing-apps-to-cloud-optimized/when-to-deploy-windows-containers-to-azure-container-service-kubernetes.md [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
+ms.date: 02/21/2022
 ---
-# When to deploy Windows Containers to Azure Container Service (that is, Kubernetes)
+# When to deploy Windows Containers to Azure Kubernetes Service (AKS)
 
-Azure Container Service optimizes the configuration of popular open-source tools and technologies specifically for Azure. You get an open solution that offers portability both for your containers and for your application configuration. You select the size, the number of hosts, and the orchestrator tools. Azure Container Service handles the infrastructure for you.
+Azure Kubernetes Service (AKS) simplifies deploying a managed Kubernetes cluster in Azure by offloading the operational overhead to Azure. You get an open solution that offers portability both for your containers and for your application configuration. You select the size, the number of hosts, and the orchestrator tools. Azure Kubernetes Service handles the infrastructure for you, it also supports Windows Server containers alongside Linux containers for more details visit this [link](https://docs.microsoft.com/azure/aks/windows-faq)
```
Comment (nishanil) [inline docs/architecture/modernize-with-azure-containers/modernize-existing-apps-to-cloud-optimized/when-to-deploy-windows-containers-to-azure-container-service-kubernetes.md:8]:
```suggestion
Azure Kubernetes Service (AKS) simplifies deploying a managed Kubernetes cluster in Azure by offloading the operational overhead to Azure. You get an open solution that offers portability both for your containers and for your application configuration. You select the size, the number of hosts, and the orchestrator tools. Azure Kubernetes Service handles the infrastructure for you. It also supports Windows Server containers alongside Linux containers. For more details on Windows OS support and the differences between Windows and Linux, read the [FAQ]https://docs.microsoft.com/azure/aks/windows-faq).
```

**Comment by VaishaliRaval** [comment]
@nishanil
Thank you for your suggestion.

File: docs/architecture/modernize-with-azure-containers/modernize-existing-apps-to-cloud-optimized/when-to-deploy-windows-containers-to-azure-container-service-kubernetes.md (hunk 1 above)
Comment (VaishaliRaval) [reply docs/architecture/modernize-with-azure-containers/modernize-existing-apps-to-cloud-optimized/when-to-deploy-windows-containers-to-azure-container-service-kubernetes.md]: @nishanil, thank you for your suggestion.

**Review by nishanil — APPROVED** [review]
LGTM! Thank You!

**Pull request merged**